cli/commands/preview/delete.ts

```diff
@@ -39,13 +40,20 @@ async function runCommand( host: string, outputFormat?: OutputFormat ): Promise<
 	}
 }
 
-export const registerCommand: RegisterCommand = ( parentCommand, rootCommand = parentCommand ) => {
-	parentCommand
-		.command( 'delete <host>' )
-		.description( __( 'Delete a preview site' ) )
-		.action( async ( host: string ) => {
-			const outputFormat = rootCommand.opts().outputFormat;
-			const normalizedHost = normalizeHostname( host );
-			await runCommand( normalizedHost, outputFormat );
-		} );
+export const registerCommand = ( yargs: Argv< GlobalOptions > ) => {
+	return yargs.command( {
+		command: 'delete <host>',
```

Comment:
```suggestion
		command: 'preview delete <host>',
```
I think it would be better if all commands were printed on the top-level help page. Having to explicitly explore the preview command to understand which commands are available will be cumbersome, especially for the first release (when there are so few commands in total).

This change would also require some changes in `src/index.ts`.

Reply:
I couldn't make it work with "2 names commands". From my testing all the commands would register as "preview", so only the last one to be registered would be available.
I still moved to the logic to the main index, grouped under the `preview` command.
I explored options for displaying sub-commands in the main help output but couldn't find a clean way to do this without manually writing the help text. 
I think that's worse than having the different helps for the sub-commands. 
What do you think?

Reply:
Ugh, pulling my hair over this issue 😭

I couldn't figure out a solution either, so let's stick with this approach for now.